src/core/CCResult.mli

```diff
@@ -188,6 +188,9 @@ end
 (** {2 Collections} *)
 
 val map_l : ('a -> ('b, 'err) t) -> 'a list -> ('b list, 'err) t
+(** [map_l f l] apply [f] on every element of [l], and, in case of success for every
+    element, return [Ok (List.map f l)]. Otherwise, it fails, and the first error is
+    return.*)
```

Comment:
"returned" :slightly_smiling_face: 
also, it's not really `Ok (List.map f l)` but the list of all the `Ok`-values, right?

Reply:
It is equivalent to `Ok (List.map f l)` but it does not call List.map. It is the list of unwrap Ok-values. But I don't know how to write it accurately and concisely.

Reply:
Explain it verbosely (I don't currently understand what the function does) and I will edit it down.

Reply:
I pushed a new version of this.